src/lightning/data/processing/data_processor.py

```diff
@@ -713,14 +713,16 @@ def _done(self, size: int, delete_cached_files: bool, output_dir: Dir) -> _Resul
             size = sum([c["dim"] if c["dim"] is not None else c["chunk_size"] for c in config["chunks"]])
             num_bytes = sum([c["chunk_bytes"] for c in config["chunks"]])
             data_format = tree_unflatten(config["config"]["data_format"], treespec_loads(config["config"]["data_spec"]))
+            num_chunks = len(config["chunks"])
+            num_bytes_per_chunk = [c["chunk_size"] for c in config["chunks"]] if num_chunks < 1024 else []
```

Comment:
Might be good to put a comment for the hardcoded 1024 limit

Reply:
done.